R/guides-none.r

```diff
@@ -0,0 +1,44 @@
+
+#' Empty guide
+#'
+#' This guide draws nothing.
+#'
+#' @inheritParams guide_axis
+#'
+#' @export
+#'
+guide_none <- function(title = waiver(), position = waiver()) {
```

Comment:
I think we should make sure this can be used for non-positional guides as well to turn them off. If I do that now I get:
```r
ggplot(mpg, aes(cty * 100, hwy * 100)) +
  geom_point(aes(colour = class)) +
  facet_wrap(vars(class)) +
  scale_color_discrete(guide = guide_none())
#> Error in if (g$order == 0) { : argument is of length zero 
```